commercetools/resource_product_type.go

```diff
@@ -167,6 +167,10 @@ func attributeTypeElement(setsAllowed bool) *schema.Resource {
 			Type:     schema.TypeString,
 			Optional: true,
 		},
+		"type_reference_id": {
```

Comment:
Can't we instead re-use the existing reference_type_id?
That would avoid confusion on the terraform provider level (altho it does not completely match the commercetools API)

Reply:
I agree! Fixed.